web-wallet/src/routes/(app)/dashboard/+layout.js

```diff
@@ -7,7 +7,7 @@ const createEmptyObject = () => ({});
 /** @type {import('./$types').LayoutLoad} */
 export async function load ({ fetch }) {
 	return {
-		currentPrice: fetch("https://api.dusk.network/v1/quote")
+		currentPrice: await fetch("https://api.dusk.network/v1/quote")
```

Comment:
That's why you had the error in the tests: I didn't want to await to avoid blocking the UI while the call resolves.